rules/gdi-structures.yaml

```diff
@@ -405,8 +405,8 @@ GAARMORY:
 		OrderName: ArtilleryStrikePowerInfoOrder
 		Prerequisites: garadr, upgrade.gdf
 		Icon: artystrike
-		ChargeTime: 270
-		QuantizedFacings: 8
+		ChargeTime: 330
+		QuantizedFacings: 2
```

Comment:
This was discused with Ganon in a Voice and yes i am sure, to make abilitys less random and more tactical. If the change sucks, we can still revert it.